src/element/types.ts

```diff
@@ -9,3 +9,13 @@ export type ExcalidrawTextElement = ExcalidrawElement & {
   actualBoundingBoxAscent?: number;
   baseline: number;
 };
+
+export type ExcalidrawArrowElement = ExcalidrawElement & {
+  type: "arrow";
+  angle: number;
+};
+
+export type ExcalidrawLineElement = ExcalidrawElement & {
+  type: "line";
+  angle: number;
+};
```

Comment:
Even though right now the type signature for `ExcalidrawArrowElement` and `ExcalidrawLineElement` look the same, I decided to keep them separate and simply doing disjunctions between the two types when waiting for an element with an `angle` property.

I don't want to early abstract the fact that they both have an `angle` for now and determine a good abstraction for angles for when we support rotation for additional shapes. 

Perhaps when #15 is implemented.